lib/service/local_notification_service.dart

```diff
@@ -0,0 +1,218 @@
+import 'dart:convert';
+
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+import 'package:myecl/service/class/message.dart' as message_class;
+import 'package:myecl/service/provider_list.dart';
+import 'package:myecl/tools/functions.dart';
+import 'package:qlevar_router/qlevar_router.dart';
+import 'package:rxdart/subjects.dart';
+import 'package:timezone/timezone.dart' as tz;
+import 'package:timezone/data/latest.dart' as tz;
```

Comment:
It is strange to import two different packages under the same name

Reply:
They are both from the same module